c/src/lib.rs

```diff
@@ -11,7 +11,6 @@
 //! C bindings to Pathfinder.
 
 use font_kit::handle::Handle;
-use foreign_types::ForeignTypeRef;
```

Comment:
Why remove this?

Reply:
Because it wasn’t being used anywhere and gave me a warning.

Is it used only on macOS maybe?  If so, I will put it back under a `#[cfg]` conditional.

Reply:
Here, I just readded it.